discord/ext/commands/cog.py

```diff
@@ -376,14 +377,29 @@ def has_error_handler(self) -> bool:
         """
         return not hasattr(self.cog_command_error.__func__, '__cog_special_method__')
 
+    @_cog_special_method
+    def cog_load(self) -> None:
+        """A special method that is called when the cog gets loaded.
+
+        This function **can** be a coroutine.
+
+        Subclasses must replace this if they want special loading behaviour.
```

Comment:
Wording here is weird, init can do special loading behaviour anyway.
Unless we want to maybe use this as an opportunity to ban touching init?

doesn't need the super call in here too.

Reply:
```suggestion
        Subclasses must replace this if they want special asynchronous loading behaviour.
        Note that the ``__init__`` special method does not allow asynchronous code to run
        inside it, thus this is helpful for setting up code that needs to be asynchronous.
```